.travis.yml

```diff
@@ -13,16 +13,11 @@ matrix:
   include:
     - env: CEPH_VERSION=luminous
```

Comment:
luminous might be end of life?
https://docs.ceph.com/docs/master/releases/schedule/

Reply:
Yeah, that's a good point. I think we'll want to drop it eventually then. However since master has been unchanged for nearly a year I want to take it a bit slow.  Since the builds with luminous are passing I think we can leave it in for a little while longer and then drop it in a separate PR, perhaps a few weeks from now.

Reply:
Ok!